resources/testdriver.js

```diff
@@ -377,13 +447,6 @@
          */
         in_automation: false,
 
-        /**
```

Comment:
Because they're on the internal methods, and are mechanical copies of the docstrings on the public API. It seems more likely to me that we'd forget to update these docstrings and so end up with something misleading than that someone would misunderstand the parameters on the implementation of the API when they're always identical to those in the interface.

Reply:
Ah, I missed that. Makes perfect sense to me if the public API has docstring, thanks :)